Gera/Ejercicios UBA/Funciones/Ejercicio12.py

```diff
@@ -0,0 +1,52 @@
+""" Contar la cantidad de palabras, separadas por uno o más espacios, de un
+telegrama que termina en punto. No utilizar funciones de string salvo la que
+indica la longitud.
+"""
+
+
+def ingresar_texto_telegrama():
+    telegrama = input("ingrese el texto del telegrama: ")
+    telegrama = telegrama.lower()
+    return telegrama
+
+
+def convertir_telegrama_en_lista(telegrama):
+    lista_telegrama = list(telegrama)
+    return lista_telegrama
+
+
+def contar_palabras_en_telegrama(lista_caracteres):
+    alfabeto = 'abcdefghijklmnñopqrstuvwxyzáéíóú'
+    palabras_contadas = 0
+    i = 0
+    p = i+1
+    while i < len(lista_caracteres)-1:
```

Comment:
Como vimos el otro día, tenés que recorrer el string entero así que un for sería más adecuado.